python/interpret_community/widget/explanation_dashboard_input.py

```diff
@@ -41,7 +41,8 @@ def __init__(
             classes=None,
             features=None,
             predict_url=None,
-            locale=None):
+            locale=None,
+            with_credentials=False):
```

Comment:
if the axios codepath in main.js works for uncredentailed scenarios (localhost) too, then we don't need this parameter

Reply:
it works in local case if with_credentials is set to false - so I could use axios only but this parameter would still be needed it seems